src/screens/Events/EventPage.js

```diff
@@ -17,8 +17,8 @@ const styles = StyleSheet.create({
     flexDirection: "row",
     justifyContent: "center",
     alignItems: "stretch",
-    backgroundColor: "#F5FCFF",
-    color: "#000000"
+    backgroundColor: colors.lightGray,
```

Comment:
At some point we should have Tim do a design pass on this app and decide on the colors for us. We're now inconsistent in that EventPage has a light grey background, but Settings has a dark grey background. Also, I grabbed the blue from our logo and while it's pretty, I'm starting to worry that it's not providing enough contrast against white for accessibility.

But, it's great that we've extracted the colors to one place!